test/transcoding/FPGALoopAttr.ll

```diff
@@ -168,6 +168,175 @@ for.end36:                                        ; preds = %for.cond29
   ret void
 }
 
+; Function Attrs: noinline nounwind optnone
```

Comment:
This test and `FPGALoopMergeInst.ll` are practically identical. I'd suggest moving these new checks to the better-documented `FPGALoopMergeInst.ll` (adding the source code to the comment section as well).

Reply:
Are you suggesting only moving new checks or duplicating is also fine (although it's obviously not)?

Reply:
My suggestion is to move the new checks; and to remove this file altogether (the latter probably deserves a separate PR so as not to confuse ourselves & the project maintainers).